app/routes/monetization/api.js

```diff
@@ -20,20 +20,18 @@ import express from 'express';
 const router = express.Router();
 const api = new MonetizationApi;
 const client = api.init();
-const base = `publications/${process.env.PUBLICATION_ID}`;
```

Comment:
Is it worth keeping this and using it as a fallback if there is no publicationId specified by the client? I know this is unlikely to happen but it might help partners navigate the code as they might prefer to set the publicationId on the server. Happy either way here so up to you!

Reply:
@ed-bird Yes, but not here. I moved the default setting to the client-side js with the default still being set by the env var.

Reply:
SG!